src/node/main.ts

```diff
@@ -12,13 +12,15 @@ import { startLink } from "./link"
 import { register } from "./routes"
 import { humanPath, isFile, loadAMDModule, open } from "./util"
 
-export const shouldSpawnCliProcess = async (args: CodeServerLib.NativeParsedArgs): Promise<boolean> => {
-  const shouldSpawn = await loadAMDModule<(argv: CodeServerLib.NativeParsedArgs) => boolean>(
-    "vs/code/node/cli",
-    "shouldSpawnCliProcess",
+export const shouldSpawnCliProcess = async (args: CodeServerLib.ServerParsedArgs): Promise<boolean> => {
+  return (
+    !args["start-server"] &&
+    (!!args["list-extensions"] ||
+      !!args["install-extension"] ||
+      !!args["install-builtin-extension"] ||
+      !!args["uninstall-extension"] ||
+      !!args["locate-extension"])
```

Comment:
nit: I don't _love_ this pattern with `!!` because I think it's harder to reason about. I also think it might be worth extracting this into a function and testing? (I may have done that in a recently merged PR).

Reply:
Funny enough, this came at straight from upstream:

https://github.com/cdr/vscode/blob/673165ca934c05d63fcab74bac7c819c32528c64/src/vs/server/main.js#L31

Reply:
Bummer! Okay oh well - something I need to nit about upstream 😂